Packs/ThreatIntelReports/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,12 @@
+
+#### Layouts
+##### Campaign Report
+- New `Permisions` section in new TIR creation with `roles` field and a disclaimer
```

Comment:
```suggestion
- Added a `Permisions` section in the new/edit layout.
```

Reply:
fixed in commit https://github.com/demisto/content/pull/16071/commits/665092f5ddcec5c34b329477590099f81ea923ff